vesting/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vesting"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+
+[dependencies]
+ink = { version = "4.2.0", default-features = false }
```

Comment:
```suggestion
ink = { version = "4.2", default-features = false }
```

Reply:
Thanks for the feedback, I will proceed with this request today evening

Reply:
done